requirements.txt

```diff
@@ -1,4 +1,4 @@
-marshmallow>=2.6
+marshmallow==3.0.0b9
```

Comment:
Don't depend strictly on 3.0.0b9 but instead >=3.0 so that consumers may use a later version of marshmallow, for example b11.